### PR TITLE
grafana: show CPU usage of Go GC (#20663)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #20663 to release-4.0

---

### What problem does this PR solve?
The existing Go GC related panels under TiDB-Runtime is hard to understand. This PR change to use GC CPU usage to show the performance impact of GC background workers.

图片

### What is changed and how it works?
Delete the `Go GC CPU` and `Go GC Count`, add a new target in `CPU usage` named `gc-cpu`. The new target represents the impact of the whole system's throughput. The higher the usage the lower the throughput.

### Release note 

- Show CPU consumed by Go GC under TiDB-Runtime.

### CODE REVIEW

File: metrics/grafana/tidb_runtime.json [hunk 1]
```diff
@@ -243,6 +243,136 @@
...
           "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "description": "TiDB cpu usage calculated with process cpu running seconds",
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "grid": {},
+<<<<<<< HEAD
```
Comment (jackysp) [inline metrics/grafana/tidb_runtime.json:254]: Please resolve conflicts.

Comment (bobotu) [reply metrics/grafana/tidb_runtime.json:254]: Need merge https://github.com/pingcap/tidb/pull/20519

**Comment by bobotu** [comment]
PTAL @jackysp @lysu

**Review by jackysp — COMMENTED** [review]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Review by tiancaiamao — APPROVED** [review]

**Pull request merged**